MediaBrowser.Api/BaseApiService.cs

```diff
@@ -299,8 +299,9 @@ protected string GetPathValue(int index)
             var first = pathInfo[0];
 
             // backwards compatibility
-            if (string.Equals(first, "mediabrowser", StringComparison.OrdinalIgnoreCase) ||
-                string.Equals(first, "emby", StringComparison.OrdinalIgnoreCase))
+            if (string.Equals(first, "mediabrowser", StringComparison.OrdinalIgnoreCase)
+                || string.Equals(first, "emby", StringComparison.OrdinalIgnoreCase)
+                || string.Equals(first, ApiEntryPoint.Instance.ConfigurationManager.Configuration.BaseUrl))
             {
                 index++;
             }
```

Comment:
Won't this break for `/baseurl/emby` now?.. maybe do something like (untested)
```cs
int urlIndex = 0;
var urlPart = pathInfo[urlIndex];

while (string.Equals(urlPart, "mediabrowser", StringComparison.OrdinalIgnoreCase) // backwards compatibility
                || string.Equals(urlPart, "emby", StringComparison.OrdinalIgnoreCase) // backwards compatibility
                || string.Equals(urlPart, ApiEntryPoint.Instance.ConfigurationManager.Configuration.BaseUrl))
{
    index++;
    urlIndex++;
    urlPart = pathInfo[urlIndex];
}
```
?

Reply:
Good catch.